ext/opcache/zend_persist_calc.c

```diff
@@ -607,7 +622,9 @@ uint32_t zend_accel_script_persist_calc(zend_persistent_script *new_persistent_s
 	zend_persist_warnings_calc(
 		new_persistent_script->num_warnings, new_persistent_script->warnings);
 
-	new_persistent_script->corrupted = 0;
+	ADD_SIZE(ZCG(mem_checksum_skip_list_count) * sizeof(zend_accel_skip_list_entry));
```

Comment:
This should lead to request of memory amount to keep uncompressed skip list together with the script. Right?

Reply:
It seems, the compression doesn't reduce SHM memory consumption.

Reply:
Actually, this is the reason of "wrong size calculation" you just found.